src/utils/image.js

```diff
@@ -219,30 +106,28 @@ export async function deleteImage(hash) {
 	const settings = await getSettings();
 	const host = await getHostSettings();
 
-	if (host === aHosts[1]) {
-		await doRequest(
-			hash,
-			'GET',
-			null,
-			null,
-			() => {},
-			() => {},
-			() => {}
-		);
-	} else {
-		await doRequest(
-			`https://api.imgur.com/3/image/${hash}`,
-			'DELETE',
-			null,
-			{
-				text: 'Authorization',
-				value: `Client-ID ${settings.apiClientId}`
-			},
-			() => {},
-			() => {},
-			() => {}
-		);
-	}
+	const get = host === aHosts[1];
+
+	await doRequest(
+		get ? hash : `https://api.imgur.com/3/image/${hash}`,
+		get ? 'GET' : 'DELETE',
+		null,
+		get
+			? null
+			: {
+					text: 'Authorization',
+					value: `Client-ID ${settings.apiClientId}`
+			  },
+		(o, res) => {
+			res();
+		},
+		(o, u, res, rej) => {
+			rej();
+		},
+		(o, u, res, rej) => {
+			rej();
+		}
+	);
```

Comment:
followup, this is a crime peepoAngry, after host modification, this needs to get rid of all of the ternaries, ideally it should just take an object whose properties are set prior, switch statement if need be